examples/vega-responsive/README.md

```diff
@@ -0,0 +1,7 @@
+[Framework examples →](../)
+
+# Vega-Lite responsive bar chart
+
+View live: <https://observablehq.observablehq.cloud/framework-example-vega-responsive/>
+
+This Observable Framework example demonstrates how to create a responsive bar chart in Vega-Lite using Framework’s `resize` built-in, which is implemented with [ResizeObserver](https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserver). The bar chart below automatically resizes to fit the container. A reusable helper function `vlresize` is provided in [`src/vlresize.js`](./src/vlresize.js).
```

Comment:
To support dark mode: 

~~~~
```js echo
import * as themes from "npm:vega-themes"
```
vl.render({
  spec:  {
    ...spec,
    config: (dark && themes.dark)
  }
})
~~~~

but it's not